### PR TITLE
Fix broken tests

### LINKED ISSUE #196: Fix broken unit tests
There have been some changes to GitHub actions that leave our UTs in this project in a broken state. See [here](https://github.com/cyberark/conjur-oss-suite-release/actions/runs/375225374) for an example.

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
- Bump Go version to 1.15
- Adds CHANGELOG to project, to track internal project changes.

### What ticket does this PR close?
Resolves #196 

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### Added
+### Changed
+- Updated Go version to 1.15. [cyberark/conjur-oss-suite-release#196](https://github.com/cyberark/conjur-oss-suite-release/issues/196)
+
+### Deprecated
+### Removed
+### Fixed
+### Security
+
```
Comment (sgnn7) [inline CHANGELOG.md:17]: We probably don't want any of these empty headers

**Review by sgnn7 — CHANGES_REQUESTED** [review]
@izgeri One small comment

**Comment by sgnn7** [comment]
I guess the tests are still broken though :(

**Comment by izgeri** [comment]
@sgnn7 unit tests are fixed up now!

**Review by sgnn7 — APPROVED** [review]

**Pull request merged**